packages/module/src/css/topology-components.css

```diff
@@ -707,6 +709,14 @@
   stroke-dashoffset: 34;
 }
 
+.pf-topology__edge__link.pf-m-drop-target {
+  stroke-dasharray: 2;
+  stroke-dashoffset: 2;
+  stroke: var(--pf-t--global--border--color--brand--default);
```

Comment:
Can we swap this token please: 
stroke: var(--pf-t--global--border--color--hover)